cocos/core/renderer/scene/skybox.ts

```diff
@@ -371,6 +406,20 @@ export class Skybox {
             this._globalDSManager.update();
         }
     }
+
+    protected _updateSubModes () {
+        if (this._model) {
+            const subModels = this._model.subModels;
+            for (let i = 0; i < subModels.length; i++) {
+                subModels[i].update();
+            }
+        }
+    }
+
+    public update () {
```

Comment:
1. Don't use such a common name `update`, use `updateXXX`, `update` have an implicit convention that it will be executed every frame.
2. Public functions should be put together